README.md

```diff
@@ -1,22 +1,151 @@
-## This project was generated with React.Js & Electron.
+# Tezster GUI
 
-Tezster-GUI comes with pretty much the same functionalities as CLI but in its beta version, it comes integrated with the CLI. After starting tezos node on your local machine with Tezster-CLI, you can clone it from here.
-To start the app in the dev environment run the following command :
+<img  src="https://tezster.s3-ap-southeast-1.amazonaws.com/TEZSTER_GUI/1_jDB5enULQVo2UfeiwD32qA.png"  alt="tezster GUI banner" align="center" />
 
-```bash
-$ npm run dev
-```
+<br />
+
+<div align="center"><strong>Start interacting with Tezos Chain within seconds</strong></div>
+
+<div align="center">Use this GUI version of Tezster which is ready-to-use Desktop app, made with React and Electron. It allows you to interact with Tezos blockchain and to deploy/test smart contracts on Tezos testnets or Sandbox node (which requires <a  href="https://docs.tezster.tech/tezster-cli">Tezster-CLI</a>).
+
+</div>
+
+<br />
+
+# Getting Started
+
+This project was generated with React.Js & Electron, Follow below steps to get started.
+
+## Prerequisites
+
+Any Operating System will work !
+
+1. Node v. 12.x+
+
+2. [Tezster-CLI](https://docs.tezster.tech/tezster-cli)
+
+## Installation Process :
+
+- Setup your Node.js environment
+
+* Install and setup [Tezster-CLI](https://docs.tezster.tech/tezster-cli)
+
+## Node.js Installation Guide
+
+Run following commands to install Node.js LTS version
+
+sudo apt-get update
+
+sudo apt-get install curl
+
+curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+
+sudo apt-get install nodejs
+
+After installing verify and check the installed version.
+
+node -v
+
+_Note: Please make sure you have the latest npm and node.js versions to be 12.x or above._
+
+## Install Tezster-CLI package
+
+[Tezster-CLI](https://docs.tezster.tech/tezster-cli) will create a sandboxed nodes in your local machine and you will be able to interact with it flawlessly with this GUI version.
+
+sudo npm install -g tezster@latest
+
+Run Following to ensure the installed version of [Tezster-CLI](https://docs.tezster.tech/tezster-cli).
+
+tezster --version
+
+To start the tester nodes on your machine run:
+
+tezster start-nodes
+
+## Tezster-GUI App Setup
+
+Inorder to start using the app we clone this repo using the following command
+
+git clone https://github.com/Tezsure/Tezster-GUI.git
+
+Now the second step is to navigate to the downloaded directory and download the required installation packages using the command
+
+sudo npm i --save
```

Comment:
why do i need to run npm with sudo 
and also why do I need to run it with --save flag?

I believe I'm commenting this 2nd time.

Reply:
electron works on top of gcc compiler which in linux and mac operating system requires access to .bin folder to install required binary files on the machine which is a system file and it requires root access.